platform/ui/src/components/measurementTable/MeasurementTable.js

```diff
@@ -75,10 +79,35 @@ class MeasurementTable extends Component {
         <ScrollableArea>
           <div>{this.getMeasurementsGroups()}</div>
         </ScrollableArea>
+        <div className="measurementTableFooter">
+          {saveFunction && (
+            <button onClick={this.saveFunction} className="saveBtn">
+              SAVE
+            </button>
+          )}
+        </div>
       </div>
     );
   }
 
+  saveFunction = async event => {
+    const { saveFunction, snackbarContext } = this.props;
+    if (saveFunction) {
+      try {
+        await saveFunction();
+        snackbarContext.show({
+          title: 'STOW SR',
+          message: 'Measurements were saved with success',
+        });
+      } catch (error) {
```

Comment:
We need to log this error. Its getting lost. I got an Error whilst saving measurements whilst testing on the deploy and couldn't debug it without pulling locally.

Reply:
This should be better now, I can logging the error.